codeserver/ubi9-python-3.11/Pipfile.lock

```diff
@@ -296,13 +304,19 @@
             ],
             "version": "==0.3.9"
         },
+        "durationpy": {
+            "hashes": [
+                "sha256:e65359a7af5cedad07fb77a2dd3f390f8eb0b74cb845589fa6c057086834dd38",
+                "sha256:fd3feb0a69a0057d582ef643c355c40d2fa1c942191f914d12203b1a01ac722a"
+            ],
+            "version": "==0.9"
+        },
         "executing": {
             "hashes": [
-                "sha256:8d63781349375b5ebccc3142f4b30350c0cd9c79f921cde38be2be4637e98eaf",
-                "sha256:8ea27ddd260da8150fa5a708269c4a10e76161e2496ec3e587da9e3c0fe4b9ab"
+                "sha256:0314a69e37426e3608aada02473b4161d4caf5a4b244d1d0c48072b8fee7bacc",
+                "sha256:19da64c18d2d851112f09c287f8d3dbbdf725ab0e569077efb6cdcbd3497c107"
             ],
-            "markers": "python_version >= '3.8'",
-            "version": "==2.1.0"
+            "version": "==1.2.0"
```

Comment:
Not immediately clear to me why this dependency got downgraded 🤔 I'm doing the `pipenv install` in a "controlled environment" (read: podman container running `base/ubi9-python-3.11` )

Was initially updated here:
![image](https://github.com/user-attachments/assets/4075ac1a-d48c-43db-a17d-201da299a4fa)

Reply:
Not clear to me either
* https://github.com/alexmojaki/executing/blob/v2.1.0/setup.cfg
* https://github.com/alexmojaki/executing/blob/v1.2.0/setup.cfg